<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Player2Window</class>
 <widget class="QDialog" name="Player2Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>448</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>742</width>
    <height>448</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>742</width>
    <height>448</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ShootButtonPl_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Shoot</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
    </font>
   </property>
   <property name="text">
    <string>Your board</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>130</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Terminal</family>
    </font>
   </property>
   <property name="text">
    <string>Enemy's board</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="MyBoard" rowstretch="0,0">
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <property name="verticalSpacing">
     <number>7</number>
    </property>
    <item row="1" column="9">
     <layout class="QVBoxLayout" name="column_10Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="column1Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="2">
     <layout class="QVBoxLayout" name="column3Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="3">
     <layout class="QVBoxLayout" name="column4Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="column2Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="8">
     <layout class="QVBoxLayout" name="column9Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="5">
     <layout class="QVBoxLayout" name="column6Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="7">
     <layout class="QVBoxLayout" name="column8Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="6">
     <layout class="QVBoxLayout" name="column7Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="1" column="4">
     <layout class="QVBoxLayout" name="column5Me">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>341</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="EnemyBoard">
    <property name="horizontalSpacing">
     <number>2</number>
    </property>
    <item row="0" column="6">
     <layout class="QVBoxLayout" name="column7Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="3">
     <layout class="QVBoxLayout" name="column4Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="column3Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="column1Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="7">
     <layout class="QVBoxLayout" name="column8Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="column2Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="9">
     <layout class="QVBoxLayout" name="column_10Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="8">
     <layout class="QVBoxLayout" name="column9Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="5">
     <layout class="QVBoxLayout" name="column6Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
    <item row="0" column="4">
     <layout class="QVBoxLayout" name="column5Enemy">
      <property name="spacing">
       <number>2</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>574</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Number of unplaced ship cells left:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Terminal</family>
          <pointsize>7</pointsize>
         </font>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>80</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_11">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>7</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="locale">
       <locale language="Ukrainian" country="Ukraine"/>
      </property>
      <property name="text">
       <string>Place ship</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
